src/carousel.tsx

```diff
@@ -114,22 +128,12 @@ const useStyles = createUseStyles((theme) => ({
         zIndex: 1,
         top: `calc(50% - ${arrowButtonSize / 2}px)`,
         '&.prev': {
-            left: -arrowButtonSize / 2,
-            [theme.mq.tabletOrSmaller]: {
```

Comment:
I think I'd keep mediaqueries and just apply overrides in desktop when inside medium/small a grid column